lib/mayaUsd/ufe/Global.cpp

```diff
@@ -108,7 +109,11 @@ MCallbackId gExitingCbId = 0;
 // Subject singleton for observation of all USD stages.
 MayaUsd::ufe::MayaStagesSubject::RefPtr g_StagesSubject;
 
-void mayaStartWaitCursor() { MGlobal::executeCommand("waitCursor -state 1"); }
+void mayaStartWaitCursor()
+{
+    ProxyRenderDelegate::setLongDurationRendering();
```

Comment:
When code somewhere creates a WaitCursor, it always sets the long duration. What happens if the action that used the wait cursor doesn't up in ProxyRenderDelegate::update()? The long duration flag will still be set.

Reply:
Any non-trivial modification to USD will trigger an update of the render delegate. If somehow we get into that situation, at worst the next update will show the wait cursor, which is benign. I just wanted to avoid toggling the wait cursor in ***all*** render delegate updates since showing the cursor requires calling commands.